readme.md

```diff
@@ -310,6 +310,25 @@ If you are simply adding additional audits/gatherers or tweaking flags, you can
 }
 ```
 
+### helpText guidelines
```

Comment:
would this be better in https://github.com/GoogleChrome/lighthouse/blob/master/CONTRIBUTING.md?